util/math.go

```diff
@@ -151,3 +152,46 @@ func CalcWork(bits uint32) *big.Int {
 	denominator := new(big.Int).Add(difficultyNum, bigOne)
 	return new(big.Int).Div(oneLsh256, denominator)
 }
+
+func getHashrate(target *big.Int, TargetTimePerBlock time.Duration) *big.Int {
```

Comment:
Please move this and GetHashrateString to a separate package, e.g. util/hashrate/hashrate.go